man/docker-cp.1.md

```diff
@@ -147,6 +148,17 @@ to a directory `/etc/my-app.d` in a container:
 The above command will copy the contents of the local `/config` directory into
 the directory `/etc/my-app.d` in the container.
 
+Third, if you want to copy a symbol link into a container, what you want is to
```

Comment:
@moxiegirl Thank you for help! :smile: 

Sorry that I just moved in another hotel with poor network connection, I'll update my PR once I get better internet connection later.